src/primary-navigation-layout/TODO.md

```diff
@@ -0,0 +1,27 @@
+# PrimaryNavigationLayout
```

Comment:
Sticking some stuff here to provide some temporary context and keep track of changes I've made for the "staging" version of the component. I will be coming back to these todos in a subsequent iteration.